prompt/prompt-llm/src/commonMain/kotlin/ai/koog/prompt/llm/OllamaModels.kt

```diff
@@ -186,7 +186,7 @@ public object OllamaModels {
          */
         public val QWQ: LLModel = LLModel(
             provider = LLMProvider.Ollama,
-            id = "qwq",
+            id = "qwq:32b",
```

Comment:
Was it a change in model name or just wrong naming?

Reply:
`qwq` does not exist in Ollama registry

Reply:
I guess we can support both `qwq` and `qwq:32b` then